lib/licensing.js

```diff
@@ -130,10 +130,67 @@ const listSubComponents = (baseDir) => {
   return subComponents
 }
 
+const listNtpBackgrounds = (metadataFile) => {
```

Comment:
this file is getting a little unwieldy, can we put these methods in a separate module? Ideally I'd like to start moving this stuff to python like we discussed for gn, but I'm ok with this as long as we split this file up into smaller chunks